corehq/apps/sms/migrations/0001_initial.py

```diff
@@ -0,0 +1,132 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from django.db import models, migrations
+import corehq.apps.sms.models
+import dimagi.utils.couch.migration
+import django.db.models.deletion
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('smsforms', '0001_initial'),
+    ]
+
+    operations = [
+        migrations.CreateModel(
+            name='MessageLogOld',
```

Comment:
Ok, I confirmed that we can remove the `MessageLogOld` model from the code and remove the references to it from `sms/admin.py`, and we can also drop the `sms_messagelog` table.  But if the table has a different owner as you mentioned then I think we might have to update the owner in all environments first otherwise dropping the table from a migration might fail.